src/resources/tech_icons.ts

```diff
@@ -3882,10 +3885,12 @@ const tech_icons = [
         path: 'https://img.shields.io/badge/Laravel-FF2D20?logo=laravel&logoColor=white&style=for-the-badge',
       },
       devicons: {
-        path: 'https://cdn.jsdelivr.net/gh/devicons/devicon/icons/laravel/laravel-plain.svg',
+        path: 'https://cdn.jsdelivr.net/gh/devicons/devicon/icons/codepen/laravel-original.svg',
         variants: [
-          'https://cdn.jsdelivr.net/gh/devicons/devicon/icons/laravel/laravel-plain.svg',
-          'https://cdn.jsdelivr.net/gh/devicons/devicon/icons/laravel/laravel-plain-wordmark.svg',
+          'https://cdn.jsdelivr.net/gh/devicons/devicon/icons/codepen/laravel-original.svg',
+          'https://cdn.jsdelivr.net/gh/devicons/devicon/icons/codepen/laravel-original-wordmark.svg',
+          'https://cdn.jsdelivr.net/gh/devicons/devicon/icons/codepen/laravel-line.svg',
+          'https://cdn.jsdelivr.net/gh/devicons/devicon/icons/codepen/laravel-line-wordmark.svg',
         ],
```

Comment:
The Laravel icons are still broken, and it seems to be due to the prefix codepen/. All other links follow the pattern brand/brand-icon.svg.

I believe it should be laravel/laravel-original.svg.

Could you please update this? Thanks!